exercises/carmen-salas/answers.js

```diff
@@ -0,0 +1,116 @@
+// Question 1
+function greet(userName) {
+  return "Good morning, " + userName + "!";
+
+}
+
+// Question 2
+function logOddNumbers() {
+  for(let i = 0; i <= 99; i++)
+  if(i % 2 ==! 0){
+    console.log(i);
+  }else{
+    
+  }
+
+}
+
+// Question 3
+function area(x, y) {
+  return (x * y); 
+}
+
+// Question 4
+function otherAngle(angle1, angle2) {
+  let angles = angle1 + angle2; 
+  return (180 - angles); 
+
+}
+
+// Question 5
+function diameterFromArea(area) {
+  let divide = (area / Math.PI);
+  let root = Math.sqrt(divide);
+  let diameter = (root * 2); 
+  return diameter;
```

Comment:
Too many variable declarations in this function. Reduce this down to no more than two lines by combining some of these statements.